proto/tendermint/consensus/types.pb.go

```diff
@@ -1,17 +1,18 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: tendermint/consensus/types.proto
```

Comment:
these are auto-generated files, we should not edit them. gofmt is ignored in these files as well. Can we revert these changes.

Reply:
done!